schemas/context/profile-individual.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-individual",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Individual Profile  Mixin",
+  "type": "object",
+  "description": "This mixin groups a set of standard fields to describe the profile of an individual.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/profile"
+  ],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-directmarketing",
+    "https://ns.adobe.com/xdm/context/profile-personal-details",
+    "https://ns.adobe.com/xdm/context/profile-owning-entities",
+    "https://ns.adobe.com/xdm/context/identitymap",
+    "https://ns.adobe.com/xdm/context/profile-preferences-details",
+    "https://ns.adobe.com/xdm/context/profile-work-details",
+    "https://ns.adobe.com/xdm/context/profile-privacy",
+    "https://ns.adobe.com/xdm/context/profile-push-details",
+    "https://ns.adobe.com/xdm/context/profile-segmentation",
+    "https://ns.adobe.com/xdm/context/profile-subscriptions"
+  ],
+  "definitions": {},
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-directmarketing"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-owning-entities"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/identitymap"
```

Comment:
identityMap is not required on an input-schema where a customer is bringing data - identity descriptors is the recommended approach -- remove from this standard mixin